File: Java/ClassesAndObjects.java
//Java Program on OOPs Basics, Class and Objects

class ClassesAndObjects
{
public static void main(String args[])
{
User user = new User();
user.displayDetails();
}
}

class User
{

String name = "Vishnu";
int age = 25;
String email = "email-id@email.com";

void displayDetails()
{
System.out.println("-- User Details --");
System.out.println("Name: "+name);
System.out.println("Age: "+age);
System.out.println("Email: "+email);
}

}

/*
Class : Class blueprint or templates for creating objects, it contains properties(variables) and behaviour(methods) that objects will have. 
        >To create a class we use "class" keyword.
        >No memory allocation for data members util objects are created.
Objects : Object is the instance of the class and can access variables and methods defined in the class.
          >To create an object we "new" keyword along with class constructor.
          >Memory is allocated when object is created.
*/

File: Kotlin/ClassesAndObjects.kt
//Kotlin program on OOPs Basics, Class and Objects

fun main()
{
val user = User() //object
user.displayDetails()
}

class User //class
{
val name = "Vishnu"
val age = 25
val email = "email-id@email.com"

fun displayDetails()
{
println("-- User Details --")
println("Name: $name")
println("Age: $age")
println("Email: $email")
}
}

/*
Class : Class blueprint or templates for creating objects, it contains properties(variables) and behaviour(methods) that objects will have. 
        >To create a class we use "class" keyword.
        >No memory allocation for data members util objects are created.
Objects : Object is the instance of the class and can access variables and methods defined in the class.
          >To create an object we call class constructor.
          >Memory is allocated when object is created.
*/